.github/workflows/ci.yml

```diff
@@ -223,9 +234,23 @@ jobs:
     secrets: inherit
     with:
      os: linux
+     runner: ubuntu-latest
      app: dl
      cluster: ${{ needs.deploy-cluster.outputs.clusterName }}
 
+  dart-tests-linux20:
```

Comment:
```suggestion
  dart-tests-linux-(ubuntu 20):
```

Reply:
Renamed but without brackets. Brackets are not allowed in the name of the job.